_inc/client/at-a-glance/scan.jsx

```diff
@@ -92,16 +93,20 @@ class DashScan extends Component {
 				if ( threats !== 0 ) {
 					return renderCard( {
 						content: [
-							<h3>{
-								__( 'Uh oh, %(number)s threat found.', 'Uh oh, %(number)s threats found.', {
-									count: threats,
-									args: { number: numberFormat( threats ) }
-								} )
-							}</h3>,
+							<h2 className="jp-dash-item__count is-warning">{ numberFormat( threats ) }</h2>,
 							<p className="jp-dash-item__description">
-								{__( '{{a}}View details at VaultPress.com{{/a}}', { components: { a: <a href="https://dashboard.vaultpress.com/" /> } } )}
-								<br />
-								{__( '{{a}}Contact Support{{/a}}', { components: { a: <a href="https://jetpack.com/support" /> } } )}
+								{ __( 'Threat found. See below for options.', 'Threats found. See below for options.', {
```

Comment:
Sorry, but this is not going to work - we need to have the number present inside the translated string. This is in order to enable translators to put the number where they need it according to language rules.
You can embed the markup inside the translated string too, that's OK.

Reply:
This PR changes the design slightly. It highlights the number before this sentence, like we're doing with the Protect module for instance. Unless I'm missing something else, @zinigor ?

Reply:
Sorry for the delay. Yes, the Protect card unfortunately has the same problem. We are trying to structure the card so that it reads as a sentence, and taking away the number, always putting it as the first word in that sentence. Here previously translators had a chance to put the number in any part of the translation according to the language rules. If we're going to emphasize the number, we either need to somehow allow text both before and after the number, or make the text read as a standalone description.
I hope this makes sense here.